packages/vifm/build.sh

```diff
@@ -2,11 +2,15 @@ TERMUX_PKG_HOMEPAGE=https://vifm.info/
 TERMUX_PKG_DESCRIPTION="File manager with vi like keybindings"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.12.1
+TERMUX_PKG_VERSION=0.13
 TERMUX_PKG_SRCURL=https://github.com/vifm/vifm/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=410b2f183e9fdefb145cf6811c106e9297c9d8f7ddfe38f8840869c63e089f95
+TERMUX_PKG_SHA256=8379397b2824cc74a91f5cfa00b5496f5d08cdcec18e3d13b64c480151225ca8
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="ncurses, file"
+# https://github.com/termux/termux-packages/issues/15889
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+ac_cv_func_getpwuid=yes
+"
```

Comment:
Do not do this for individual package *unnecessarily*.

Reply:
Rather `getpwuid()` should be disabled in this package for backward-compatibility.

Reply:
Note that in #15889 there is no conclusion as to what is the expected behavior. There is a probability that disabling `getpwuid()` is.